src/app/domains/MediaLibrary/combined/MediaLibraryModal/MediaLibraryModal.template.js

```diff
@@ -34,8 +34,8 @@ function MediaLibraryModal(props) {
     setData(COLLECTIONS?.MEDIA, mediaId, data)
   }
 
-  const currentQuestion = useFormContext()
-  const dispatch = useFormContextDispatch()
+  const currentQuestion = useCurrentQuestionContext()
+  const dispatch = useCurrentQuestionContextDispatch()
```

Comment:
Please, provide more semantic and meaningfull variable name here... (Imagine situation when you are importing one ore two more contexts with\or their dispatch actions...)